src/scripts/main.js

```diff
@@ -355,6 +355,21 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+const visiblePeople = people.map((person) => ({
+  name: person.name,
+  sex: person.sex,
+  born: person.born,
+  died: person.died,
+  age: person.died - person.born,
+  century: Math.ceil(person.died / 100),
+}));
```

Comment:
This code is creating a new array `visiblePeople` that is a copy of `people` with additional properties. This is not efficient because it creates unnecessary duplication of data. Instead of creating a new array, you could add the new properties directly to the objects in the `people` array. This can be done using the `forEach` method instead of `map`.